packages/synapse-react-client/src/components/RecentPublicationsGrid/RecentPublicationsGrid.tsx

```diff
@@ -20,6 +27,14 @@ type PublicationCardProps = {
   pub: Row
 }
 
+const convertDoiToLink = (doi: string) => {
+  doi = doi.trim()
+  if (DOI_REGEX.test(doi)) {
```

Comment:
Did we have a utility function for this?

Reply:
It looks like we have similar logic in GenericCard.tsx and CardFooter.tsx, you are welcome to refactor to create this utility.